tests/utils/model_configs.py

```diff
@@ -538,6 +538,115 @@ def _update_and_add_testing_config(
     },
 )
 
+_update_and_add_testing_config(
+    # Tests logit distillation.
+    "mistral",
+    "mistral_distill_logits",
+    updates={
+        ("model", "base_model", "head", "distillation_model"): "teacher",
+        ("reference_models"): {
+            "teacher": {
+                "model": {
```

Comment:
You'll need complete model descriptions, ex. copied from another, otherwise the created model will be too big.